src/main/Resources/traffic.csv

```diff
@@ -0,0 +1,13 @@
+Car, WantsToDrive

```

Comment:
@nicolaiparlog why has this to be lower case? For linux it matters, but why should it be all lower case?